src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,36 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.health = 100;
+    this.name = name;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = true;
```

Comment:
if you once hid this beast, you could not change it in the future. Think how to make it work both ways

Reply:
![image](https://github.com/mate-academy/js_herbivores_and_carnivores/assets/107731333/e3d173c3-a4f2-4fdd-b129-66ceb8cad8eb)

Reply:
Sorry, my mistake)

Reply:
i can add new function unhide or unhide beast when he was attacked, but his status was `hidden`.
Also, there is zero information about changing property `hidden` backwards, so just read the task and comment after it :)